Failed: InactiveEntityPassivationSpec

https://jenkins.akka.io:8498/job/akka-artery-tcp-cluster-tests/514/testReport/junit/akka.cluster.sharding/InactiveEntityPassivationSpec/Passivation_of_inactive_entities_must_passivate_entities_when_they_haven_t_seen_messages_for_the_configured_duration/

```
Error Message

java.lang.AssertionError: assertion failed: received unexpected message RealMessage(1 passivating,Actor[akka://InactiveEntityPassivationSpec/system/sharding/myType/1/1#1403380725]) after 966 millis

Stacktrace

sbt.ForkMain$ForkError: java.lang.AssertionError: assertion failed: received unexpected message RealMessage(1 passivating,Actor[akka://InactiveEntityPassivationSpec/system/sharding/myType/1/1#1403380725]) after 966 millis
	at akka.testkit.TestKitBase.expectNoMsg_internal(TestKit.scala:696)
	at akka.testkit.TestKitBase.expectNoMessage(TestKit.scala:661)
	at akka.testkit.TestKitBase.expectNoMessage$(TestKit.scala:660)
	at akka.testkit.TestKit.expectNoMessage(TestKit.scala:896)
	at akka.cluster.sharding.InactiveEntityPassivationSpec.$anonfun$new$2(InactiveEntityPassivationSpec.scala:89)
	at akka.cluster.sharding.InactiveEntityPassivationSpec$$Lambda$1368/2105805110.apply(Unknown Source)
```

--- a/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/InactiveEntityPassivationSpec.scala
+++ b/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/InactiveEntityPassivationSpec.scala
@@ -53,6 +53,8 @@ object InactiveEntityPassivationSpec {
 class InactiveEntityPassivationSpec extends AkkaSpec(InactiveEntityPassivationSpec.config) {
   import InactiveEntityPassivationSpec._
 
+  val smallTolerance = 300.millis
+
   "Passivation of inactive entities" must {
 
     "passivate entities when they haven't seen messages for the configured duration" in {
@@ -86,7 +88,8 @@ class InactiveEntityPassivationSpec extends AkkaSpec(InactiveEntityPassivationSp
 
       // make sure "1" hasn't seen a message in 3 seconds and passivates
       val timeSinceOneSawAMessage = (System.nanoTime() - timeOneSawMessage).nanos
-      probe.expectNoMessage(3.seconds - timeSinceOneSawAMessage)
+      val timeUntilPassivate: FiniteDuration = (3.seconds - timeSinceOneSawAMessage) - smallTolerance
+      probe.expectNoMessage(timeUntilPassivate)
       probe.expectMsg("1 passivating")
 
       // but it can be re activated just fine:

